README.md

```diff
@@ -62,9 +79,19 @@ The scope required for sending metrics is the `Ingest metrics` scope in the **AP
 
 The `prefix` parameter specifies an optional prefix, which is prepended to each metric key, separated by a dot (`<prefix>.<namespace>.<name>`).
 
-#### Default Labels/Dimensions
+#### Default Dimensions
+
+The `default_dimensions` parameter can be used to optionally specify a list of key/value pairs, which will be added as additional dimensions to all data points.
+Dimension keys are unique, and labels on instruments will overwrite the default dimensions if key collisions appear.
+
+#### Export OneAgent Metadata
+
+If running on a host with a running OneAgent, setting the `export_oneagent_metadata` option to `True` will export metadata collected by the OneAgent to the Dynatrace endpoint.
+If no Dynatrace API endpoint is set, the default exporter endpoint will be the OneAgent endpoint, and this option will be set automatically.
+Therefore, if no endpoint is specified, we assume a OneAgent is running and export to it, including metadata.
 
-The `tags` parameter can be used to optionally specify a list of key/value pairs, which will be added as additional labels/dimensions to all data points.
+##### Dimensions precedence
+When specifying default dimensions, labels and OneAgent metadata enrichment, the precedence of dimensions with the same key is as follows: Default dimensions are overwritten by labels passed to instruments, which in turn are overwritten by the OneAgent dimensions (even though the likeliness of a collision here is very low). If the same key is present _within_ any of the three categories, the last occurrence of each key will be used.
```

Comment:
```suggestion
When specifying default dimensions, labels and OneAgent metadata enrichment, the precedence of dimensions with the same key is as follows:
Default dimensions are overwritten by labels passed to instruments, which in turn are overwritten by the OneAgent dimensions
(even though the likeliness of a collision here is very low, since the OneAgent metadata only contains [Dynatrace reserved dimensions](https://www.dynatrace.com/support/help/how-to-use-dynatrace/metrics/metric-ingestion/metric-ingestion-protocol/#syntax) starting with `dt.*`).
```